src/components/GrantCard.tsx

```diff
@@ -0,0 +1,215 @@
+import { useState } from 'react';
+import {
+  Alert,
+  Image,
+  Modal,
+  Pressable,
+  StyleSheet,
+  Text,
+  TouchableHighlight,
+  View,
+} from 'react-native';
+import Icon from '../../assets/icons';
+import ViewContainer from './ViewContainer';
+
+const styles = StyleSheet.create({
+  container: {
+    height: 128,
+    justifyContent: 'space-between',
+  },
+  centeredView: {
+    flex: 1,
+    justifyContent: 'center',
+    alignItems: 'center',
+    marginTop: 22,
+  },
+  cardContainer: {
+    flexDirection: 'column',
+    alignItems: 'flex-start',
+    backgroundColor: '#C94420',
+    width: 330,
+    height: 140,
+    borderRadius: 13,
+    justifyContent: 'space-between',
+    marginBottom: '8%',
+    shadowColor: 'black',
+    shadowOpacity: 0.2,
+    shadowRadius: 2,
+    shadowOffset: { width: 2, height: 4 },
+    display: 'flex',
+  },
+  whiteText: {
+    color: 'white',
+    margin: '2%',
+    marginLeft: '4%',
+    textAlign: 'left',
+    justifyContent: 'space-between',
+    fontSize: 15,
+  },
+  modalBodyText: {
+    color: '#171717',
+    margin: '2%',
+    marginLeft: '4%',
+    textAlign: 'left',
+    justifyContent: 'space-between',
+    fontSize: 15,
+  },
+  titleText: {
+    color: 'white',
+    margin: '3%',
+    marginTop: '6%',
+    textAlign: 'left',
+    fontSize: 15,
+  },
+  closeButtonText: {
+    color: 'white',
+    margin: '2%',
+    marginLeft: '4%',
+    textAlign: 'center',
+    justifyContent: 'space-between',
+    fontSize: 15,
+  },
+  modalView: {
+    margin: 20,
+    backgroundColor: 'white',
+    borderRadius: 13,
+    width: 340,
+    padding: 30,
+    shadowColor: '#000',
+    shadowOffset: {
+      width: 0,
+      height: 2,
+    },
+    shadowOpacity: 0.25,
+    shadowRadius: 4,
+    elevation: 5,
+  },
+  button: {
+    borderRadius: 5,
+    padding: 10,
+    elevation: 2,
+  },
+  buttonOpen: {
```

Comment:
Remove -- this is unused

Reply:
Before making a PR (or as you go) for frontend work, I'd always do a final check that all the css classes you introduce are actually used (`cmd + F`-ing all of them should do the trick). In many cases you'll introduce new classes as you go but you don't end up needing them later, so make sure you clean them up!